src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/FhirToHL7v2/ADT_A08/ADT-A08-01-expected.hl7

```diff
@@ -1,5 +1,5 @@
 
 MSH|^~\&|||||08/18/1989 03:11:00|||MSG00001|P^^|2.8^^
 EVN||08/18/1989 03:11:00
-PID|1||||EVERYMAN&&&&^ADAM^A^III^Dr.^MD^D^^^^^19241012^^PF^Addsm~Josh&&&&^stanley^^^^^D^^^^^19241010^19241015^^|SMITH^Angela^L||M||2106-3^White^HL70005~2028-9^Asian^HL70005|1000 Hospital Lane^Ste. 123^Ann Arbor ^MI^99999^USA^^^^^^^20000110^20000120||2222^^^^^^^^^^^~12121212^^^^^^^^^^^~8484^^^^^^^^^^^~717171^^^^^^^^^^^|7777^^^^^^^^^^^~1111^^^^^^^^^^^~89898989^^^^^^^^^^^~021212^^^^^^^^^^^|ara^Arabic^HL70296^^^v2~eng^English^HL70296^^^v2.1|M^Married|AME|4000776^^^AccMgr&2.4&ISO^VN|||1212121^^^NTH&rt23&HCD^AND^^19241011^19241012
+PID|1||PATID1234^^^&b3ad3a9f-d4f6-4638-0a09-2da771733488&GUID^MR^^^^^^^~123456789^^^USSSA&&^SS^^^^^^^~PNM1234^^^&1fca4c8f-7562-b755-9a05-5f0f49096e5b&GUID^MR^^19241011^19241012^^^^||EVERYMAN&&&&^ADAM^A^III^Dr.^MD^D^^^^^19241012^^PF^Addsm~Josh&&&&^stanley^^^^^D^^^^^19241010^19241015^^|SMITH^Angela^L||M||2106-3^White^HL70005~2028-9^Asian^HL70005|1000 Hospital Lane^Ste. 123^Ann Arbor ^MI^99999^USA^^^^^^^20000110^20000120||2222^^^^^^^^^^^~12121212^^^^^^^^^^^~8484^^^^^^^^^^^~717171^^^^^^^^^^^|7777^^^^^^^^^^^~1111^^^^^^^^^^^~89898989^^^^^^^^^^^~021212^^^^^^^^^^^|ara^Arabic^HL70296^^^v2~eng^English^HL70296^^^v2.1|M^Married|AME|4000776^^^AccMgr&2.4&ISO^VN|||1212121^^^NTH&rt23&HCD^AND^^19241011^19241012
```

Comment:
PID.3.4 here doesn't look right to me but I'm not sure.

Theirs:
`test1&2.16.1&HCD`

Yours:
`&b3ad3a9f-d4f6-4638-0a09-2da771733488&GUID`

Reply:
Looking closer, it looks like you might need to search the patientOrganizations for those GUIDs. Those organization objects contain the rest of your values, like I showed yesterday with `patientMotherIdentifier`

Reply:
Nothing in the AZDO repo is relevant here.

Reply:
Okay, it should match now but I don't think it's pretty.